hack/make-release.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+
+release::replace_runner_images() {
+  local -r tag="$1"
+
+  local manifests="$(find ./manifests/implementation/runner -iname "*.yaml" -a -not -path "*cloudsql*")"
+
+  for manifest in $manifests; do
+    sed -i.bak -E "s|^(              image: ghcr.io/capactio/)(.+):(.+)|\1\2:${tag}|" "${manifest}"
```

Comment:
there still may be a problem when previous image is set to `pr` 
As you can see, we need to remove in such case also `pr` 
```diff
            -  image: ghcr.io/capactio/pr/terraform-runner:PR-407
            +  image: ghcr.io/capactio/pr/terraform-runner:123456
```